### ISSUE TITLE
Error loading (amqp) plugin

### ISSUE THREAD

**Issue by Cashlix** [guardian/alerta#230]
Hi Nick,

in `f9dc85 "Add plugin name for better error msgs"` you're passing the plugin name to the plugin when instanciating.
```
                ---    plugins.append(plugin())
                +++   plugins.append(plugin(ep.name))
```
The explicit ctor of the amqp plugin doesn't expect a second parameter and thus crashes upon instanciation.

May be also other plugins are affected.

Best,
Felix.

**Comment by satterly** [comment]
Thanks for reporting this. I'll update Pypi with a new release shortly.

**Comment by Cashlix** [comment]
Thanks for quick response!

**Comment by satterly** [comment]
This is available now as alert server version 4.7.30
